Readme.md

```diff
@@ -105,6 +126,7 @@ Attribution 4.0 International License][cc-by-license].
 [nuget]: https://www.nuget.org/
 [travis]: https://travis-ci.org/ForNeVeR/1969
 [visual-studio]: https://www.visualstudio.com/
+[jetbrains-rider]: https://www.jetbrains.com/rider/
```

Comment:
I keep the links section in alphabetical order. Please move that link to the proper position.